waspc/examples/todoApp/src/server/serverSetup.js

```diff
@@ -5,7 +5,8 @@ let someResource = undefined
 
 export const getSomeResource = () => someResource
 
-const setup = async () => {
+const setup = async (app, _server) => {
```

Comment:
I would recommend adding a type for the `ServerSetupFn` like:

```ts
type ServerSetupFn = (context: ServerSetupFnContext) => void;

type ServerSetupFnContext = {
  app: express.Application,
  server: http.Server,
};
```
which the user can import like `import { ServerSetupFnContext } from '@wasp/types';` or similar.

Reply:
Yep, can do. 👍🏻 This is just a WIP skeleton at the moment but will beef it out shortly before tagging for review.

Reply:
Added some types, thanks!